crates/driver/src/domain/competition/auction.rs

```diff
@@ -413,6 +413,22 @@ impl Price {
     pub fn in_eth(self, amount: eth::TokenAmount) -> eth::Ether {
         (amount.0 * self.0 .0 / Self::BASE).into()
     }
+
+    /// Convert an amount of ETH into a token amount using this price.
+    ///
+    /// Converting 1 ETH into GNO
+    ///
+    /// # Examples
+    /// ```
```

Comment:
I think we have to set up the compilation of doc tests, right?
Without that these examples might get out of date very fast.

Reply:
https://github.com/cowprotocol/services/pull/2675